text/0000-programmatic-style-injection.md

```diff
@@ -98,13 +98,19 @@ Programmatically importing a stylesheet module from JavaScript using an import d
 
 However, until now, the compile-time behavior and runtime behavior for stylesheet modules have never been fully specified. This section fills this specification gap.
 
-Stylesheets modules are identifiable by their `.css` extension. This kind of module only exposes a single default export. The exported value is an opaque object, in other words, the internal implementation of the stylesheet is not exposed to developers. Keeping the exported value opaque is necessary until all the browser vendors agree on [Constructable Stylesheets](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleSheet) implementation.
+Stylesheets modules are identifiable by their `.css` extension. This kind of module only exposes a single default export. The exported value is an opaque object, in other words, the internal implementation of the stylesheet is not exposed to developers. Developers shouldn't rely on the shape of those objects as it may change.
 
 ```js
 import stylesheetA from './styles-a.css';
 import stylesheetB from './styles-b.css';
 ```
 
+Scoped stylesheet modules work similarly than standard stylesheet modules. They can be imported using the `.scoped.css` extension.
+
+```js
+import scopedStylesheet from './styles.scoped.css';

```

Comment:
@pmdartus Re-evaluating this: I think approach # 2 would be problematic for making scoped styles default in light DOM (https://github.com/salesforce/lwc-rfcs/pull/65).

If scoped styles are the default for light DOM, then `foo.css` means "scoped," and the only way to communicate that is with the query param. So I think we should instead keep the query param, and transform JS files to append `?scoped=true` to imports of `*.scoped.css`. The fact that the import occurred in a user-authored JS file is our signal that the file should be treated as scoped.

Unfortunately, there is now a weird situation like this:

```
- foo.js
- foo.css
- foo.html (light DOM)
```

In the above scenario, `foo.css` is treated as scoped when imported by `foo.html`, and treated as unscoped when imported by `foo.js`.

Maybe this isn't such a big deal, though, because people will probably not name it `foo.css` alongside `foo.html` and then import from `foo.js`? If they're doing programmatic style injection, they can name the CSS file whatever they want, because it's not implicitly tied to the HTML file.

/cc @divmain